packages/backend/src/utils/downloader.ts

```diff
@@ -150,6 +160,16 @@ export class Downloader {
           return;
         }
 
+        if (checkSum) {
+          const actualSha = checkSum.digest('hex');
+          if (this.sha256 !== actualSha) {
+            callback({
```

Comment:
is the file deleted if it does not match ?

Reply:
I think it is not atm. Maybe i should 👍

Reply:
yes, I think we should consider the sha as part of the download process, meaning it is simply failed, if the sha is not matching.